packages/schematics/angular/migrations/update-8/update-lazy-module-paths.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
```

Comment:
Nit: is this tslint disable required here?

This package has a direct dependency on `@angular-devkit/schematics` which should make this redundant.